accesscontrol/jwt.go

```diff
@@ -89,33 +87,33 @@ func NewJWT(options *JWTOptions) (*JWT, error) {
 	}
 
 	if options.Key != "" && options.KeyFile != "" {
-		return nil, confErr.Message("key and keyFile provided")
+		return nil, fmt.Errorf("key and keyFile provided")
 	}
 
 	key := []byte(options.Key)
 	if options.KeyFile != "" {
 		k, err := readKeyFile(options.KeyFile)
 		if err != nil {
-			return nil, confErr.With(err)
+			return nil, err
```

Comment:
Wie würde man das machen, wenn man hier auch ein bisschen Kontext (z.B. "error reading key file") geben wollte?

Reply:
```suggestion
			return nil, confErr.Label("jwt read error").Messagef("could not read from file: %s", options.Keyfile).With(err)
```

Reply:
Ich vermute, dass du dann das "configuration error" doppelt bekommst: einmal hiervon, einmal aus server.go

Reply:
```
configuration error: my-jwt: loading jwt definition failed: configuration error
```

Reply:
Ohne weiteren Kontext wäre es
```
configuration error: my-jwt: loading jwt definition failed: open /path/to/non-existing/file: no such file or directory
```

Reply:
Just return the error on callee side without wrapping.
Can you explain the reason why the error types are moved from the implementation side to configuration code?
Additionally @alex-schneider is working on it with #242 .

Reply:
The issue with inner errors was initially solved in #225. I have added this lines in bae47e22489799975726bc7c83a3ca2ec5ea8475 to the master branch.